src/sidebar/services/test/search-filter-test.js

```diff
@@ -57,6 +57,21 @@ describe('sidebar.search-filter', () => {
       assert.equal(result.any[2], 'hi-fi');
       assert.equal(result.any[3], 'a:bc');
     });
+
```

Comment:
nb. `services/search-filter.js` is the query parser that turns strings into objects representing the query. `services/view-filter.js` is the query executor that filters annotations against a query object.